src/proc/hole-filling-filter.cpp

```diff
@@ -53,7 +53,6 @@ namespace librealsense
                                                << "Unsupported mode for hole filling selected: value " << val
                                                << " is out of range." );
 
-            std::lock_guard<std::mutex> lock(_mutex);
```

Comment:
Can you explain?

Reply:
No need to lock for access to a `uint8_t` variable. Access is atomic in HW.